Automattic/wp-calypso
# Add 2 year breakdown to Akismet checkout

## Proposed Changes

* Generalize the filenames of the updated 2 year breakdown design to encompass more than just Jetpack
* Show updated breakdown of 2 year plans on Akismet checkout

## Testing Instructions

1. Checkout this branch or use the Calypso Live link
2. Checkout this diff on WPCOM: D125628-code
3. In your 0-sandbox.php file, add the following line
```
lang=php
define( 'USE_STORE_SANDBOX', true );
```
4. Go to any of the Akismet 1 year plans or monthly plans in checkout (ex. `/checkout/akismet/ak_plus_yearly_2`)
5. Make sure the 2 year breakdown shows up and that it correctly switches the plan to the bi-yearly plan which the CTA is clicked
![image](https://github.com/Automattic/wp-calypso/assets/65001528/be74d8fe-be94-48da-89c1-2cea868f7757)
![image](https://github.com/Automattic/wp-calypso/assets/65001528/c9d65d57-d6d3-4963-9c76-240d515927c0)

## Pre-merge Checklist
